NPM package has unspecified dependency on @openzeppelin/contracts

The `@openzeppelin/wizard` package has a devDependency on `@openzeppelin/contracts`.

However, [`print-versioned.ts`](https://github.com/OpenZeppelin/contracts-wizard/blob/master/packages/core/src/print-versioned.ts) has an import `from "@openzeppelin/contracts/package.json"`

When trying to run a project that consumes `@openzeppelin/wizard`, the following error occurs unless that consuming project has a dependency on `@openzeppelin/contracts`.
```
node:internal/modules/cjs/loader:1075
  const err = new Error(message);
              ^

Error: Cannot find module '@openzeppelin/contracts/package.json'
Require stack:
- /Users/eric/git/create-web3-dapp/node_modules/@openzeppelin/wizard/dist/print-versioned.js
- /Users/eric/git/create-web3-dapp/node_modules/@openzeppelin/wizard/dist/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/eric/git/create-web3-dapp/node_modules/@openzeppelin/wizard/dist/print-versioned.js:4:24)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/eric/git/create-web3-dapp/node_modules/@openzeppelin/wizard/dist/print-versioned.js',
    '/Users/eric/git/create-web3-dapp/node_modules/@openzeppelin/wizard/dist/index.js'
  ]
}
```

--- a/packages/core/CHANGELOG.md
+++ b/packages/core/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- Remove unspecified dependency on `@openzeppelin/contracts`. ([#233](https://github.com/OpenZeppelin/contracts-wizard/pull/233))
+
 ## 0.2.0 (2022-11-08)
 
 - Reduce default block time to 12 seconds in `governor`. ([fdcf912](https://github.com/OpenZeppelin/contracts-wizard/commit/fdcf9129354692b3b7e0fa694233fdd62a1e99bb))

--- a/packages/core/src/print-versioned.ts
+++ b/packages/core/src/print-versioned.ts
@@ -1,11 +1,11 @@
-import { version as contractsVersion } from "@openzeppelin/contracts/package.json";
+import contracts from '../openzeppelin-contracts';
 import type { Contract } from "./contract";
 import { printContract } from "./print";
 
 export function printContractVersioned(contract: Contract): string {
   return printContract(contract, {
     transformImport: p =>
-      p.replace(/^@openzeppelin\/contracts(-upgradeable)?/, `$&@${contractsVersion}`),
+      p.replace(/^@openzeppelin\/contracts(-upgradeable)?/, `$&@${contracts.version}`),
   });
 }
 

